lib/committee/request_unpacker.rb

```diff
@@ -3,11 +3,12 @@ class RequestUnpacker
     def initialize(request, options={})
       @request = request
 
-      @allow_form_params  = options[:allow_form_params]
-      @allow_query_params = options[:allow_query_params]
-      @coerce_form_params = options[:coerce_form_params]
-      @optimistic_json    = options[:optimistic_json]
-      @schema             = options[:schema]
+      @allow_form_params   = options[:allow_form_params]
```

Comment:
What do you think about having `allow_header_params` on by default? As you noticed, it seems to be part of the spec, and the reason that `allow_query_params` and `allow_form_params` exist is that they're doing something that's a little abnormal from the standard Committee conventions. Header parameters are almost entirely orthogonal to other types of parameters, and I suspect it's fine just to always allow them.

Reply:
Thank you for your pointing it out.
I think `allow_header_params` should be true by default too. 
In the same way as other parameter, I'll set default parameter on `RequestValidation`'s constructor.